### ISSUE TITLE
Change the Characters that are having the Coin split applied

### ISSUE THREAD

**Issue by Janiax** [fantasycalendar/FoundryVTT-ItemPiles#137]
**Is your feature request related to a problem? Please describe.**
I had a character assigned to the GM so I could use it as a Narrator, and when splitting the loot in an item pile it always splits with that GM character as well.

**Describe the solution you'd like**
It would be nice to have a Setting which shows the player characters that are included in the loot split and edit which will be included.

**Describe alternatives you've considered**
I have currently solved it by unassigning the character from my GM

**Additional context**
This would also be really good when a player is controlling more than one character

**Comment by 9ryex5** [comment]
Also, it would be cool if the default was to split among online players that own an actor and not all players that own an actor.

**Comment by Haxxer** [comment]
> Also, it would be cool if the default was to split among online players that own an actor and not all players that own an actor.

You can toggle this off in the sharing settings.

**Comment by TheGreatRedDuke** [comment]
I also would love to see this addition. Having the ability white and/or black list characters/tokens would be really helpful. I often have players that control more than one token and the "pet/summon/drone" that they additionally control always throws off the split currency calculations.

**Comment by Haxxer** [comment]
Split currency only splits between the user's assigned characters, ie the ones they have chosen to embody through their user in the lower left. If that is not the case, you are describing a bug @TheGreatRedDuke.

**Comment by matthewbstroud** [comment]
I agree this would be a nice feature, in my games I have done filtering based on the folder an actor is in.  So, I keep on the _real_ player characters in an actor folder called Players.  It might be easy to add a filter where it is a drop-down that let's you select the folder.

**Comment by WerepyrexLestat** [comment]
I have an issue with splitting currency, it is adding the GM to the active players list, and I cannot turn it off

**Comment by Haxxer** [comment]
> I have an issue with splitting currency, it is adding the GM to the active players list, and I cannot turn it off

Check if the GM has a character assigned to them in the player list config.
